src/blocks/disk_space.rs

```diff
@@ -208,10 +208,10 @@ impl Block for DiskSpace {
         // Send percentage to alert check if we don't want absolute alerts
         let alert_val = if self.alert_absolute {
             (match self.unit {
-                Prefix::Tera => result << 40,
-                Prefix::Giga => result << 30,
-                Prefix::Mega => result << 20,
-                Prefix::Kilo => result << 10,
+                Prefix::Tera => result >> 40,
```

Comment:
Maybe it's better to leave `result` as bytes and shift the warning/alert values to bytes instead. For example, with this way a `result` of 4.6GB ends up becoming just `4`

Reply:
You are right. Or just convert to float and divide.

Reply:
Yeah either way